projects/packages/sync/src/modules/class-woocommerce.php

```diff
@@ -242,6 +243,38 @@ public function filter_order_item( $args ) {
 		return $args;
 	}
 
+	/**
+	 * Handler for filtering out non-whitelisted order item meta.
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @param array $args Hook arguments.
+	 * @return array|false False if not whitelisted, the original hook args otherwise.
+	 */
+	public function filter_meta( $args ) {
+		if (
```

Comment:
We could add an extra check here to ensure `$args[2]` is set:
```suggestion
		if ( empty( $args[2] ) ) {
		    return false;
		if (
```

Reply:
Good idea. 
I've added it within the same check:
```
	if (
			! empty( $args[2] ) && $this->is_whitelisted_order_item_meta( $args[2] )
		) 
```
I notice `filter_meta` in both `class-comments.php` and `class-posts.php` do not check for `$args[2]` before using it, so perhaps we ought to follow up with those as well.